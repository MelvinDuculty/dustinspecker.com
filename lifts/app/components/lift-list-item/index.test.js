import LiftListItem from '.'
import React from 'react'
import renderer from 'react-test-renderer'

test('LiftListItem', () => {
  const props = {
    id: '73',
    name: 'squats',
    workWeight: 300
  }

  const component = renderer.create(
    <LiftListItem {...props} />
  )

  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
