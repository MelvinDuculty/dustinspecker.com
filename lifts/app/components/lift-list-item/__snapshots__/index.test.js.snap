// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiftListItem 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation1-4 MuiPaper-rounded-2"
>
  <a
    className="MuiButtonBase-root-38 MuiListItem-root-28 MuiListItem-default-31 MuiListItem-gutters-35 MuiListItem-button-36"
    href="#/73"
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
  >
    <div
      className="MuiListItemText-root-41 MuiListItemText-inset-42"
    >
      <h3
        className="MuiTypography-root-47 MuiTypography-subheading-54 MuiListItemText-primary-44"
      >
        Squats
      </h3>
      <p
        className="MuiTypography-root-47 MuiTypography-body1-56 MuiTypography-colorTextSecondary-69 MuiListItemText-secondary-45"
      >
        300 lbs
      </p>
    </div>
    <span
      className="MuiTouchRipple-root-71"
    />
  </a>
</div>
`;
